docs/topics/security.rst

```diff
@@ -37,10 +39,17 @@ valid domains as the second argument::
                     ...
                 ])
             ),
-            ["goodsite.com", "*.goodsite.com"],
+            [".goodsite.com", "http://.goodsite.com:80", "http://other.site.com"],
         ),
     })
 
+Note: If you want to resolve any domains, just add ``["*"]`` as second argument.
```

Comment:
"any domains, just add * as second argument" -> "any domain, then use the origin `*`"